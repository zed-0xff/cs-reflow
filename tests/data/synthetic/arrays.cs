void case1()
{
    var a = new int[] { 1, 2, 3 };
    if (a.Length == 3)
    {
        called1();
    }

    if (a[1] == 2)
    {
        called2();
    }

    a[1] = 123;
    if (a[1] == 123)
    {
        called3();
    }
}

void case2()
{
    var a = new int[3];
    if (a.Length == 3)
    {
        called1();
    }

    if (a[1] == 0)
    {
        called2();
    }

    a[1] = 123;
    if (a[1] == 123)
    {
        called3();
    }
}
