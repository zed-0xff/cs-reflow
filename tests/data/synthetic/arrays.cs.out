void case1()
{
    called1();
    called2();
    called3();
}

void case2()
{
    called1();
    called2();
    called3();
}
